aspnetcore/native-aot/index.md

```diff
@@ -0,0 +1,180 @@
+---
+title: ASP.NET Core support for .NET native AOT
+author: mitchdenny
+description: Learn about ASP.NET Core support for .NET native AOT
+monikerRange: '>= aspnetcore-8.0'
+ms.author: midenn
+ms.custom: mvc
+ms.date: 04/10/2023
+uid: native-aot/index
+---
+# ASP.NET Core support for .NET native AOT.
+
+ASP.NET Core 8.0 introduces support for [.NET native ahead-of-time (AOT)](/dotnet/core/deploying/native-aot/). Not all ASP.NET Core features are compatible with native AOT at this time. 
+
+* How to get started.
+* The benefits and potential drawbacks.
+* The limitations.
+
+## Getting started with .NET native AOT deployment in ASP.NET Core
+
+When developing an app to be published using native AOT, a developer writes their application normally. Native AOT compilation occurs when an app is published, and the native AOT compiler analyses the code in the app to identify coding patterns incompatible with AOT.
```

Comment:
@MichalStrehovsky  The following is your excellent suggest with the MS style guide applied. No future tense. Correct sentences (remove `-`), avoid parenthetical expressions. Split up long sentences.

```suggestion
Native AOT is a publishing option. AOT compilation happens when the app is published. A project opted into Native AOT publishing still uses JIT for F5 debugging, dotnet run, or dotnet build. There are some observable differences:
* Some run time features that aren't compatible with Native AOT are disabled and throw exceptions at run time.
* A source analyzer is enabled to highlight code that isn't compatible with Native AOT. At publish time, the entire app, including NuGet packages, are analyzed for compatibility again.
```